src/physics/nonlinear_solid.cpp

```diff
@@ -228,7 +232,28 @@ void NonlinearSolid::defineInputFileSchema(std::shared_ptr<axom::inlet::SchemaCr
   table->addDouble("tz", "Cantilever tip traction in the z direction.")->defaultValue(0.0);
 
   auto solver_table = table->addTable("solver", "Linear and Nonlinear Solver Parameters.");
-  serac::EquationSolver::defineInputFileSchema(solver_table);
+  serac::EquationSolver::defineInputFileSchema(*solver_table);
 }
 
 }  // namespace serac
+
+using serac::NonlinearSolid;
+
+NonlinearSolid FromInlet<NonlinearSolid>::operator()(axom::inlet::Table& base)
```

Comment:
What is the argument for using these templated builder methods as opposed to passing an inlet table to the constructor?

Reply:
They are less intrusive (require no changes to the class definition) which makes it possible to define "deserialization" from third-party types (MFEM, STL, etc).  `nlohman::json` uses a similar approach, as does the STL for `std::hash`.